tests/Generators/GoogleGeneratorTest.php

```diff
@@ -35,4 +35,12 @@ public function it_correctly_generates_all_day_events_by_dates(): void
             $this->generator()->generate($this->createEventMultipleDaysViaStartEndWithTimezoneLink())
         );
     }
+
+    /** @test */
+    public function it_correctly_generates_all_day_events_by_dates_diff_tz(): void
```

Comment:
I think having this test is beneficial for all generators. How about moving it to the `GeneratorTestContract`?

Reply:
Agreed!
Maybe also moving the ones I added as well?